torch/csrc/jit/script/compiler.cpp

```diff
@@ -3402,6 +3402,9 @@ void runCleanupPasses(std::shared_ptr<Graph>& to_clean) {
   // remove any uses of tuples that we inserted that are not needed
   LowerSimpleTuples(to_clean);
   ConstantPooling(to_clean);
+  // run constant propagation on immutable types only instead of full constant
+  // prop so we do not jitter
```

Comment:
"so we do not jitter" doesn't really have enough explanation here to understand it. It will be really easy to cargo-cult on this comment later (e.g. also feel like this pass is needed long after it really was because this comment doesn't describe what jitters).  Why does constant prop cause jitter when done completely but not when done to immutable things? Is it only because we haven't created a complicated enough test that only works with types like integers and tuples?